pkg/products/monitoring/reconciler.go

```diff
@@ -211,7 +211,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 
 	isMultiAZCluster, err := resources.IsMultiAZCluster(ctx, serverClient)
 	if err != nil {
-		r.Log.Error("Error when deciding if the cluster is multi-az or not. Defaulted to false:", err)
+		r.Log.Warning("Error when deciding if the cluster is multi-az or not. Defaulted to false: " + err.Error())
```

Comment:
Remove the word `Error` from the warning message. This was one of the issues with the word "error" showing up in greps of the logs.

Reply:
for `Log.Error` it shows and `ERRO` message. 
For example: `ERRO[2021-03-15T13:55:16Z] Rollout system-app deployment                 error="deploymentconfigs.apps.openshift.io \"system-app\" not found" product=3scale
` 
So for the grep it should be fine

Reply:
That would be correct but if someone is using grep incorrectly for example `grep -i 'erro' log.txt` This would also show up this warning message as it starts with "Error".

Reply:
Oh, yes, then it definitely should be changed. Will `failure` fit in here? Technically: Error -> Fault -> Failure and only the "Failure" we can observe.